stubs/console-menu/consolemenu/items/function_item.pyi

```diff
@@ -0,0 +1,23 @@
+from collections.abc import Callable
+from typing import Any
+
+from consolemenu.console_menu import ConsoleMenu
+from consolemenu.items import ExternalItem as ExternalItem
+
+class FunctionItem(ExternalItem):
+    function: Callable[..., Any]
+    args: list[Any]
+    kwargs: dict[str, Any]
+    return_value: Any | None
+    def __init__(
+        self,
+        text: str,
+        function: Callable[[Any], Any],
+        args: list[Any] | None = ...,
+        kwargs: dict[str, Any] | None = ...,
```

Comment:
I'd type these two as Sequence and Mapping instead. (Particularly for `args`, it's useful to allow tuples.)

Reply:
For clarification, `args` should be sequence and `kwargs` should be Mapping?